src/test/javascript/spec/service/apollon-diagram.service.spec.ts

```diff
@@ -1,147 +1,135 @@
 import { ApollonDiagram } from 'app/entities/apollon-diagram.model';
 import { UMLDiagramType } from 'app/entities/modeling-exercise.model';
 import { HttpResponse } from '@angular/common/http';
-import * as chai from 'chai';
-import sinonChai from 'sinon-chai';
-
-chai.use(sinonChai);
-const expect = chai.expect;
 
 const resourceUrl = SERVER_API_URL + 'api';
 
 describe('ApollonDiagramService', () => {
     let courseId: number;
     let apollonDiagram: ApollonDiagram;
+    let apollonDiagramService: ApollonDiagramService;
+    let httpTestingController: HttpTestingController;
     beforeEach(() => {
         TestBed.configureTestingModule({
             imports: [HttpClientTestingModule],
             providers: [ApollonDiagramService],
-        });
+        })
+            .compileComponents()
+            .then(() => {
+                apollonDiagramService = TestBed.inject(ApollonDiagramService);
+                httpTestingController = TestBed.inject(HttpTestingController);
+            });
         courseId = 1;
         apollonDiagram = new ApollonDiagram(UMLDiagramType.ClassDiagram, courseId);
     });
 
-    it('should be initialized', inject([ApollonDiagramService], (apollonDiagramService: ApollonDiagramService) => {
-        expect(apollonDiagramService).to.be.ok;
-    }));
-
-    it('should create a diagram', fakeAsync(
-        inject([ApollonDiagramService, HttpTestingController], (apollonDiagramService: ApollonDiagramService, server: HttpTestingController) => {
-            // Set up
-            const url = `${resourceUrl}/course/${courseId}/apollon-diagrams`;
-            const responseObject = apollonDiagram;
+    it('should create a diagram', fakeAsync(() => {
+        // Set up
+        const url = `${resourceUrl}/course/${courseId}/apollon-diagrams`;
+        const responseObject = apollonDiagram;
 
-            let response: HttpResponse<ApollonDiagram>;
+        let response: HttpResponse<ApollonDiagram>;
 
-            apollonDiagramService.create(apollonDiagram, courseId).subscribe((receivedResponse: HttpResponse<ApollonDiagram>) => {
-                response = receivedResponse;
-            });
+        apollonDiagramService.create(apollonDiagram, courseId).subscribe((receivedResponse: HttpResponse<ApollonDiagram>) => {
+            response = receivedResponse;
+        });
 
-            const requestWrapper = server.expectOne({ url });
-            requestWrapper.flush(responseObject);
+        const requestWrapper = httpTestingController.expectOne({ url });
+        requestWrapper.flush(responseObject);
 
-            tick();
+        tick();
 
-            expect(requestWrapper.request.method).to.equal('POST');
-            expect(response!.body).to.equal(responseObject);
-            expect(response!.status).to.equal(200);
-        }),
-    ));
+        expect(requestWrapper.request.method).toBe('POST');
+        expect(response!.body).toBe(responseObject);
```

Comment:
If the reference necessarily has to be the same here, `toBe()` is fine. If you want to compare the state of the two objects, I would suggest using `toEqual()`.